airflow/jobs/scheduler_job_runner.py

```diff
@@ -1562,9 +1570,9 @@ def _fail_tasks_stuck_in_queued(self, session: Session = NEW_SESSION) -> None:
             for ti in tasks_stuck_in_queued:
                 if repr(ti) in cleaned_up_task_instances:
                     self._task_context_logger.warning(
-                        "Marking task instance %s stuck in queued as failed. "
+                        "Marking task instance %s stuck in queued for {} seconds as failed. "
```

Comment:
It may be worth mentioning why a task stuck in queued (i.e. `task_queued_timeout`) and possibly even linking to the config docs for `task_queued_timeout`